CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
-### 1.9.0 (Next)
+### 2.0.0 (Next)
 
 #### Features
 
 * [#2353](https://github.com/ruby-grape/grape/pull/2353): Added Rails 7.1 support - [@ericproulx](https://github.com/ericproulx).
 * [#2355](https://github.com/ruby-grape/grape/pull/2355): Set response headers based on Rack version - [@schinery](https://github.com/schinery).
 * [#2360](https://github.com/ruby-grape/grape/pull/2360): Reduce gem size by removing specs - [@ericproulx](https://github.com/ericproulx).
+* [#2361](https://github.com/ruby-grape/grape/pull/2361): Remove rack::auth::digest - [@ninoseki](https://github.com/ninoseki).
```

Comment:
quote and capitalize to match the actual class name, `Rack::Auth::Digest`.

Reply:
Did it by [1310747](https://github.com/ruby-grape/grape/pull/2361/commits/1310747db21ae53d4ace475d2fde5ca97e617edd) and [3125460](https://github.com/ruby-grape/grape/pull/2361/commits/31254606d51508c44809361a98a955988620c1f9). (Sorry for dirty commit history)